Makefile

```diff
@@ -43,16 +43,16 @@ lint:
 
 build: build-operator build-csv-merger build-webhook
 
-build-operator: $(SOURCES) ## Build binary from source
+build-operator: $(SOURCES) ## Build HCO binary from source from source under cmd/hyperconverged-cluster-operator
```

Comment:
I'd remove `from source under cmd/hyperconverged-cluster-operator` since that is only the entry point and it could make more confusion than help